services/core-web/src/styles/tools/syncfusion.scss

```diff
@@ -30,16 +30,55 @@ $base-font: #000000;
 $error-font: $error-color; //#f44336;
 
 @import "../../../node_modules/@syncfusion/ej2-base/styles/material.scss";
+@import "../../../node_modules/@syncfusion/ej2-buttons/styles/material.scss";
+@import "../../../node_modules/@syncfusion/ej2-dropdowns/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-icons/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-inputs/styles/material.scss";
+@import "../../../node_modules/@syncfusion/ej2-navigations/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-popups/styles/material.scss";
-@import "../../../node_modules/@syncfusion/ej2-buttons/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-splitbuttons/styles/material.scss";
-@import "../../../node_modules/@syncfusion/ej2-navigations/styles/material.scss";
+@import "../../../node_modules/@syncfusion/ej2-lists/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-layouts/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-grids/styles/material.scss";
+@import "../../../node_modules/@syncfusion/ej2-react-pdfviewer/styles/material.scss";
 @import "../../../node_modules/@syncfusion/ej2-react-filemanager/styles/material.scss";
+// @import "../../../node_modules/@syncfusion/ej2-react-documenteditor/styles/material.scss";
+// @import "../../../node_modules/@syncfusion/ej2-react-spreadsheet/styles/material.scss";
```

Comment:
remove?

Reply:
Added a comment to explain why we may need it in the future, if that's ok with you